job_tracker/views.py

```diff
@@ -112,29 +112,49 @@ def job_search(request):
         job_data.append(info)
     return JsonResponse(job_data, safe=False)
 
+def contact_index(request):
+    contact = Contact.objects.filter(user=request.User)
```

Comment:
I think this should be called 'contacts', otherwise the thing you are passing the view something that doesn't exist.